### PR TITLE
Courses Page documentation

### PR DESCRIPTION
**Pull request by thallada**
Also fix unrelated sphinx errors related to headings in the Enrollment Geography
page.

FYI @katymyw

### CODE REVIEW

**Review by dsjen — APPROVED** [review]
👍

File: docs/en_us/dashboard/source/Courses_Page.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+* Course ID
+* Course Start Date
+* Course End Date
+* Total Enrollment
+* Current Enrollment
+* Change Last Week
+* Verified Enrollment
+
+.. image:: images/course_list.png
+ :alt: Screenshot of the Insights Courses page table with five example courses.
```
Comment (srpearce) [inline docs/en_us/dashboard/source/Courses_Page.rst]: I'd take out "Screenshot of".

File: docs/en_us/dashboard/source/Courses_Page.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+To see the insights view for a specific course, click the course name and ID.
```
Comment (srpearce) [inline docs/en_us/dashboard/source/Courses_Page.rst]: Capitalize Insights

File: docs/en_us/dashboard/source/Courses_Page.rst [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+You can then access more data for that course. 
+
+For detailed information about the computations, see :ref:`Enrollment
+Computations`.  Note that the enrollment data that the computations include is
+the same as the summary metrics presented in the :ref:`Enrollment_Activity`
+report.
+
+.. note::
+ On Edge, the summary table will not include course name, start date, or end
```
Comment (srpearce) [inline docs/en_us/dashboard/source/Courses_Page.rst]: ... table does not include the course name...

**Review by srpearce — APPROVED** [review]

**Comment by srpearce** [comment]
@thallada Looks great! Can you squash your commits?

Also, I'm not sure if the test failure is something you want to look into. Will it be OK for me to merge if that test is still failing?

**Comment by thallada** [comment]
@srpearce it looks like we are having some unrelated problems with Code Cov. We don't block merging on Code Cov failures anyways. Assuming that the travis pr/push builds that are running right now succeed, then this should be good to merge!

I don't know how to publish the new HTML to the insights documentation website. I'm assuming you will do that once this PR is merged?

**Comment by srpearce** [comment]
@thallada Yep, I'll merge this PR and then build the guide so you can see the changes.

**Pull request merged**